### PR TITLE
Add new versions for Microsoft.Datadog RP

### PR DESCRIPTION
**Pull request by vikotha**
# Choose a PR Template

Switch to "Preview" on this description then select one of the choices below.

<a href="?expand=1&template=data_plane_template.md">Click here</a> to open a PR for a Data Plane API.

<a href="?expand=1&template=control_plane_template.md">Click here</a> to open a PR for a Control Plane (ARM) API.

### CODE REVIEW

**Comment by vikotha** [comment]
Related to previous PR: https://github.com/Azure/azure-rest-api-specs/pull/31053

File: specification/datadog/resource-manager/readme.md [hunk 1]
```diff
@@ -26,7 +26,23 @@ These are the global settings for the Datadog API.
 title: Microsoft Datadog Client
 openapi-type: arm
 openapi-subtype: rpaas
-tag: package-2023-01
+tag: package-2023-10
+```
+### Tag: package-2023-01
```
Comment (mikekistler) [inline specification/datadog/resource-manager/readme.md:31]: This looks like a copy/paste error:
```suggestion
### Tag: package-2023-10
```

Comment (vikotha) [reply specification/datadog/resource-manager/readme.md]: Updated for both new versions

**Comment by nikhgup** [comment]
@vikotha please fix the issue related to apiview failure

**Comment by vikotha** [comment]
> @vikotha please fix the issue related to apiview failure

@nikhgup, seems to be a false alarm. No where in the swagger we are referencing to the path "examples/CreationSupported_List", but it's still looking for one. Does operationId name affects it?
We do not see this failure in private repo for this version.

Can you help from where it is looking for the path?

**Review by nikhgup — APPROVED** [review]
approving as discussed offline. api- version sync from private repo

**Comment by vikotha** [comment]
/pr merge
We have not added our project to code Owners yet, hence need someone to merge this.

**Review by dipeshbhakat-microsoft — APPROVED** [review]
approved

**Pull request merged**